dev/Breadcrumb/BreadcrumbBarItem.cpp

```diff
@@ -344,6 +345,35 @@ void BreadcrumbBarItem::OpenFlyout()
     }
 }
 
+winrt::Style BreadcrumbBarItem::RetrieveFlyoutStyle()
+{
+    if (const auto& grid = m_grid.get())
+    {
+        const auto& resources = grid.Resources();
+
+        /*
+        if (resources.HasKey(box_value(L"DefaultBreadcrumbBarItemFlyoutPresenterStyle")))
+        {
+            const auto& value = resources.Lookup(box_value(L"DefaultBreadcrumbBarItemFlyoutPresenterStyle"));
+            return value.try_as<winrt::Style>();
+        }
+        */
+
+        for (const auto& themeDictionaryEntry : resources)
```

Comment:
This looks very hacky. Is there a reason why we dont have the flyout on the button in markup ?

Reply:
last time I tried I remember having the issue of the flyout showing up when I clicked the breadcrumb item, while this shouldn't matter for most buttons it would matter in the last button, I can do some testing tomorrow so I can verify if this behaviour is still happening

Reply:
turns out setting the flyout style in the markup works as it should so that was changed